### PR TITLE
Add linting 

### PR DESCRIPTION
**Pull request by adericbourg**

### CODE REVIEW

**Comment by adericbourg** [comment]
Mmh, linter and formatter disagree. Nice.

File: .travis.yml [hunk 1]
```diff
@@ -10,8 +10,12 @@ go:
 
 go_import_path: github.com/criteo/graphite-remote-adapter
 
+install:
+- go get github.com/golang/lint/golint
+
 script:
 - make style test
+- make lint
```
Comment (Thib17) [inline .travis.yml]: What's the difference with `make style` ?

File: client/graphite/paths/cache.go [hunk 2]
```diff
@@ -11,6 +11,7 @@ var (
 	pathsCacheEnabled = false
 )
 
+// InitPathsCache inits paths for the cache.
```
Comment (Thib17) [inline client/graphite/paths/cache.go]: /paths for the cache/cache for the paths/

File: client/graphite/paths/write.go [hunk 3]
```diff
@@ -13,11 +13,12 @@ import (
 	"github.com/prometheus/common/model"
 )
 
+// ToDatapoints builds points from
```
Comment (Thib17) [inline client/graphite/paths/write.go]: incomplete comment: `from samples`

File: .travis.yml (hunk 1 above)
Comment (adericbourg) [reply .travis.yml]: `make style` uses `fmt`: it is only about code formatting.

`go lint` should detect violations from [Effective Go](https://golang.org/doc/effective_go.html) and [CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments).

File: client/graphite/paths/cache.go (hunk 2 above)
Comment (adericbourg) [reply client/graphite/paths/cache.go]: Done.

File: client/graphite/paths/write.go (hunk 3 above)
Comment (adericbourg) [reply client/graphite/paths/write.go]: Done

**Pull request merged**